Marlin/src/gcode/control/M605.cpp

```diff
@@ -56,7 +56,26 @@
     planner.synchronize();
 
     if (parser.seen('S')) {
+      DualXMode  previous_mode, requested_mode = (DualXMode)parser.value_byte();
+      previous_mode = dual_x_carriage_mode;
       dual_x_carriage_mode = (DualXMode)parser.value_byte();
+      scaled_duplication_mode = false;
+
+      if (dual_x_carriage_mode == DXC_SCALED_DUPLICATION_MODE) {
+        if (previous_mode != DXC_DUPLICATION_MODE) {
+          SERIAL_ECHOPGM("Printer must be in DXC_DUPLICATION_MODE prior to \n");
+          SERIAL_ECHOPGM("specifying DXC_SCALED_DUPLICATION_MODE.\n");
+          dual_x_carriage_mode = DEFAULT_DUAL_X_CARRIAGE_MODE;
+          return;
+        }
+        float x_jog = current_position[X_AXIS] - .1;
+        scaled_duplication_mode = true;
+        stepper.set_directions();
+        planner.buffer_line(x_jog, current_position[Y_AXIS], current_position[Z_AXIS], current_position[E_AXIS], feedrate_mm_s, 0);
+        x_jog += .1;
+        planner.buffer_line(x_jog, current_position[Y_AXIS], current_position[Z_AXIS], current_position[E_AXIS], feedrate_mm_s, 0);
```

Comment:
Maybe we already have a method to accomplish what this is doing, or if needed we can add one to the Planner.

Reply:
Actually....   I think I can delete that.  But the current code is well tested so I left it alone.
The next Pull Request will most likely have that job back and forth removed.